src/base/static/components/organisms/filter-menu.js

```diff
@@ -0,0 +1,210 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from "prop-types";
+import { NavButton, CloseButton } from "../molecules/buttons";
+import { Image } from "../atoms/imagery";
+import { Link, RegularTitle, LargeLabel } from "../atoms/typography";
+import styled from "react-emotion";
+import mq from "../../../../media-queries";
+import { connect } from "react-redux";
+import {
+  updateFilters,
+  filtersPropType,
+  filtersSelector,
+} from "../../state/ducks/filters";
+import {
+  placeFormsConfigPropType,
+  placeFormsConfigSelector,
+} from "../../state/ducks/forms-config";
+
+import Modal from "react-modal";
+Modal.setAppElement("#main");
+
+const FilterNavButton = styled(linkProps => (
+  <NavButton variant="raised" color="primary" onClick={linkProps.onClick}>
+    {linkProps.children}
+  </NavButton>
+))(() => ({
+  [mq[0]]: {
+    display: "none",
+  },
+  [mq[1]]: {
+    display: "block",
+  },
+}));
+
+const FilterMenuWrapper = styled("div")({
+  display: "flex",
+  flexDirection: "column",
+  width: "100%",
+  height: "100%",
+});
+
+const FilterMenuHeading = styled("div")({
+  display: "flex",
+});
+const FilterMenuTitle = styled(RegularTitle)({
+  textAlign: "center",
+  marginLeft: "auto",
+  marginRight: "auto",
+});
+const FilterOptions = styled("div")({
+  marginLeft: "auto",
+  marginRight: "auto",
+});
+const CategoryFilterOption = styled("div")(props => ({
+  display: "flex",
+  flexDirection: "row",
+  marginBottom: "8px",
+  alignItems: "center",
```

Comment:
I'd suggest the filter options have a little padding so they're not so crowded when highlighted

```suggestion
  alignItems: "center",
  padding: "4px 8px 4px 8px",
```